component/src/main/ballerina/net.ftp/ftp_client_endpoint.bal

```diff
@@ -0,0 +1,55 @@
+package ballerina.net.ftp;
+
+////////////////////////////
+/// FTP Client Endpoint ///
+///////////////////////////
+
+@Description {value:"Represents an FTP client endpoint"}
+@Field {value:"config: The configurations associated with the endpoint"}
+public struct ClientEndpoint {
```

Comment:
Shouldn't we give specific names? Should we call this FTPClientEndpoint?

Reply:
This will be renamed  as an FtpClient according to the new naming convention.